Enable "translations" feature to add the translations from the fluent repos.

--- rslib/Cargo.toml.orig	2021-05-04 05:04:50 UTC
+++ rslib/Cargo.toml
@@ -7,6 +7,11 @@ license = "AGPL-3.0-or-later"
 description = "Anki's Rust library code"
 build = "build/main.rs"
 
+[features]
+default = ["translations"]
+
+translations = []
+
 [lib]
 name = "anki"
 path = "src/lib.rs"
